src/python/pants/reporting/zipkin_reporter.py

```diff
@@ -25,13 +33,14 @@ def __init__(self, endpoint):
   def get_max_payload_bytes(self):
     return None
 
-  def send(self, payload):
+  def send(self, file_path):
     try:
-      requests.post(
-        self.endpoint,
-        data=payload,
-        headers={'Content-Type': 'application/x-thrift'},
-      )
+      command = 'curl -v -X POST -H "Content-Type: application/json" --data @{} {}'.format(file_path, self.endpoint)
+      args = command.split(' ')
```

Comment:
I suspect you don't actually want to split on spaces, because you have some quoted things. Maybe just make your `list` manually:
```python
command = [
  "curl",
  "-v",
  "-X", "POST",
  "-H", "Content-Type: application/json",
  ...
]
```